mmv1/products/memorystore/Instance.yaml

```diff
@@ -301,74 +301,77 @@ properties:
     description: "Endpoints for the instance."
     output: true
     item_type:
-      type: Array
-      description: "A group of PSC connections. They are created in the same VPC network, one for each service attachment in the cluster."
-      item_type:
-        type: NestedObject
-        properties:
-          - name: 'pscConnectionId'
-            type: String
-            description:
-              "Output only. The PSC connection id of the forwarding rule connected
-              to the\nservice attachment. "
-            output: true
-          - name: 'ipAddress'
-            type: String
-            description:
-              "Output only. The IP allocated on the consumer network for the
-              PSC forwarding rule. "
-            output: true
-          - name: 'forwardingRule'
-            type: String
-            description: "Output only. The URI of the consumer side forwarding rule.\nFormat:\nprojects/{project}/regions/{region}/forwardingRules/{forwarding_rule} "
-            output: true
-          - name: 'projectId'
-            type: String
-            description:
-              "Output only. The consumer project_id where the forwarding rule is
-              created from. "
-            output: true
-          - name: 'network'
-            type: String
-            description:
-              "Output only. The consumer network where the IP address resides, in
-              the form of\nprojects/{project_id}/global/networks/{network_id}. "
-            output: true
-          - name: 'serviceAttachment'
-            type: String
-            description:
-              "Output only. The service attachment which is the target of the PSC connection, in the form of
-              projects/{project-id}/regions/{region}/serviceAttachments/{service-attachment-id}."
-            output: true
-          - name: 'pscConnectionStatus'
-            type: Enum
-            description:
-              "Output Only. The status of the PSC connection: whether a connection exists and ACTIVE or it no longer exists.
-              \n Possible values:\n ACTIVE \n NOT_FOUND"
-            output: true
-            enum_values:
-              - 'ACTIVE'
-              - 'NOT_FOUND'
-          - name: 'connectionType'
-            type: Enum
-            description:
-              "Output Only. Type of a PSC Connection.
-              \n Possible values:\n CONNECTION_TYPE_DISCOVERY \n CONNECTION_TYPE_PRIMARY \n CONNECTION_TYPE_READER"
-            output: true
-            enum_values:
-              - 'CONNECTION_TYPE_READER'
-              - 'CONNECTION_TYPE_PRIMARY'
-              - 'CONNECTION_TYPE_DISCOVERY'
+      type: NestedObject
```

Comment:
Could you set up some time with me on GVC. This reads to me like a breaking change. We are changing existing field structure. I can't find this field documented externally on the [rest API](https://cloud.google.com/memorystore/docs/redis/reference/rest/v1beta1/projects.locations.instances).

Reply:
Actually, this field is not used a.t.m. There is no change in the proto. The previous person who added this field in Terraform had made a mistake in exposing this, but because the feature is not launched yet(its behind mendel), nobody was affected.
Please see this field in the proto, its unchanged since when it was added.
https://source.corp.google.com/piper///depot/google3/google/cloud/memorystore/v1main/memorystore.proto;l=687

This is for a new feature. The docs for it will only be live on launch day.

Reply:
ah got it! We'll need to hold off on merging until the feature is live. Thanks for explaining that.

If possible could we have a handwritten test that checks the presence of one or more of these fields. Modifying an existing test would also be okay.